x/ibc/07-tendermint/types/header.go

```diff
@@ -48,35 +51,17 @@ func (h Header) ValidateBasic(chainID string) error {
 	if h.ValidatorSet == nil {
 		return sdkerrors.Wrap(clienttypes.ErrInvalidHeader, "validator set is nil")
 	}
+	if !bytes.Equal(h.ValidatorsHash, h.ValidatorSet.Hash()) {
+		return sdkerrors.Wrap(clienttypes.ErrInvalidHeader, "validator set does not match hash")
+	}
+	if !bytes.Equal(h.NextValidatorsHash, h.NextValidatorSet.Hash()) {
+		return sdkerrors.Wrap(clienttypes.ErrInvalidHeader, "next validator set does not match hash")
+	}
 	return nil
 }
 
 // ToABCIHeader parses the header to an ABCI header type.
 // NOTE: only for testing use.
 func (h Header) ToABCIHeader() abci.Header {
```

Comment:
We can remove this util and use the `tmtype` function directly

Reply:
Since updated.